### ISSUE TITLE
Generated URLs miss IIS sub-application prefix

### ISSUE THREAD

**Issue by rducom** [LuccaSA/RestDrivenDomain#294]
Missing IIS sub-application prefix

base URI : https://micheletaugustin.local/popleetalent/api/ReviewsCampaigns/4d0e37e1-0eeb-4390-55dd-08d655dc68c7
generated navigation URI : https://micheletaugustin.local/api/ReviewsCampaigns/4d0e37e1-0eeb-4390-55dd-08d655dc68c7

![image](https://user-images.githubusercontent.com/5228175/50238087-cc6ee500-03be-11e9-9754-fab0e6f9a2bb.png)

**Comment by Poltuu** [comment]
yep, need to implement your own urlprovider for this, as it is not the classic case

**Comment by rducom** [comment]
I believe we should be able to extract this

**Comment by nfaugout** [comment]
For me it depens on how we retrieve the information "https://micheletaugustin.local".

If we get that information within Rdd, directly from the HttpContext, then we sould go further and retrieve the first URL segment as well.

If we ask the consumer project to pass this information to Rdd through settings for instance, then we should ask it for the non mandatory first segment.
